tests/test_pytest/bad_plugins/product_templates/bad_150H_spec_types.yaml

```diff
@@ -0,0 +1,22 @@
+interface: product_templates
```

Comment:
Note we currently only have "bad" test plugins for inter_alg_cmap - will delay adding additional bad pluguins until product interface is finalized.